test/DynamoCoreWpfTests/NodeViewCustomizationTests.cs

```diff
@@ -58,7 +58,8 @@ public void Watch3DHasViewer()
             }
 
             Open(@"UI\CoreUINodes.dyn");
-
+            //TODO why did this pass before?
```

Comment:
Anything pending to investigate here?

Reply:
nope, this seems fine, must have temporarily broken something as I was in progress.